app/utils/monitoring.client.tsx

```diff
@@ -0,0 +1,31 @@
+import { useLocation, useMatches } from '@remix-run/react'
+import * as Sentry from '@sentry/remix'
+import { useEffect } from 'react'
+
+
+export function init() {
+	Sentry.init({
+		dsn: ENV.SENTRY_DSN,
+		integrations: [
+			new Sentry.BrowserTracing({
+				routingInstrumentation: Sentry.remixRouterInstrumentation(
+					useEffect,
+					useLocation,
+					useMatches,
+				),
+			}),
+			// Replay is only available in the client
+			new Sentry.Replay(),
+		],
+
+		// Set tracesSampleRate to 1.0 to capture 100%
+		// of transactions for performance monitoring.
+		// We recommend adjusting this value in production
```

Comment:
Can you elaborate on this? What's the recommended value? Why don't we set this to the recommended value?

Reply:
@kentcdodds Honestly, I just copied it from the default onboarding but yea we can change this. The right amount depends on your own needs but I think 10% is a reasonable default.